packages/fast-components-react-msft/src/call-to-action/call-to-action.props.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import { IManagedClasses, IMSFTCallToActionClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+export enum CallToActionAppearance {
+    lightweight= "lightweight",
+    primary= "primary",
+    secondary= "secondary"
+}
+
+export interface ICallToActionHandledProps {
+
+    /**
+     * The call to action appearance
+     */
+    appearance?: CallToActionAppearance;
+
+    /**
+     * The content
+     */
+    children?: React.ReactNode | React.ReactNode[];
+
+    /**
+     * The destination address
+     */
+    href: string;
+}
+
+export interface ICallToActionUnhandledProps extends React.AllHTMLAttributes<HTMLElement> {}
```

Comment:
If using Hypertext, this should be HTMLAnchorElement. If we were using the `<Button />` component this would be accurate.